spring-integration-ip/src/test/java/org/springframework/integration/ip/tcp/TcpSendingMessageHandlerTests.java

```diff
@@ -619,7 +648,9 @@ public void run() {
 				}
 			}
 		});
-		AbstractConnectionFactory ccf = new TcpNetClientConnectionFactory("localhost", port);
+		assertTrue(latch.await(10, TimeUnit.SECONDS));
```

Comment:
The `assert` with `latch` has been moved here, but isn't removed from the old place like it is in other tests.
It doesn't hurt because for the second time it is counted down already, but it should be removed to avoid confuse in the future.
Will be done on merge. :smile: